docs/device_reference.md

```diff
@@ -2,16 +2,18 @@
     - Fire OS 5: Based on Android 5.1 (Lollipop, API level 22)
     - Fire OS 6: Based on Android 7.1 (Nougat, API level 25)
 
-There are currently two devices:
+There are currently three devices:
```

Comment:
nit: maybe we should just remove this line so we don't have to keep updating it

Reply:
Fair.  It just duplicates the table anyway.